README.md

```diff
@@ -204,6 +204,10 @@ For example, see the "lc_subject", "composer", and "rights_holder" fields in [th
 (https://github.com/curationexperts/alexandria-v2/blob/master/spec/fixtures/csv/pamss045_with_local_authorities.csv).
 
 
+# Caveats
+
+* Reindexing all objects (to an empty solr) requires two passes.  The first pass will guarantee that the collections are indexed, and the second pass will index the collection name on all the objects. The object indexer looks up the collection name from solr for speed.
```

Comment:
All of those will work, but the foremost is the case I was considering:  when there are no records in Solr, then you must reindex two times.  Your last two cases won't work when there are no records in Solr, because AF::Base#all reads from Solr and would return an empty array.